packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx

```diff
@@ -46,9 +45,7 @@ function ChainCard(chain: ChainData) {
 
       <div className="message">
         <RenderMarkdown source={message} noMargin />
-        {error ? (
-          <a href={path.join(apiUrl, dnpName, "logs")}>More info</a>
-        ) : null}
+        {error ? <Link to={`/packages/${dnpName}/logs`}>More info</Link> : null}
```

Comment:
Check other parts of the UI how to import the `/packages` path from pages/packages/data. Also use urlJoin

Reply:
done in last commit